UnifiedMap/UnifiedMap/UnifiedMap.cs

```diff
@@ -33,6 +33,7 @@ public class UnifiedMap : View
         /// <summary>
         /// The bindable pin callout tapped command property.
         /// </summary>
+        [Obsolete("This method will soon be deprecated. Use InfoWindowClicked event instead.")]
```

Comment:
What do you think is better naming that works for both platforms? `InfoWindow` or `PinCallout` consistency would be nice and i would actually prefer `PinCallout` since its a bit more obvious what it means. An `InfoWindow` does not inherently say that it's on a Pin.

Reply:
Apple calls it a Callout, but UnifiedMaps is cross platform, it's not always best to default on one side and renaming it is a "breaking" change.

So, we have `Pins`, we have `Overlays` and therefore my suggestion would be `PinInfoView` Which makes it clear that the InfoView is for Pins. If Overlays get something similar then it would be `OverlayInfoView`

Reply:
I totally agree.